packages/testharness/src/pageObjects/wait.js

```diff
@@ -1,3 +1,5 @@
+import AbortController from 'abort-controller-es5';
```

Comment:
In my opinion, any packages that you publish that are used in WC also need to be reviewed by a team member. The method right now allows you to push huge amounts of un-reviewed code into our project.